### DISCUSSION TITLE
Direction of induced velocity of rotor in hover

### DISCUSSION THREAD

**Discussion by christianhauschel** [byuflowlab/FLOWUnsteady#153] [Q&A]
I am simulating a rotor in hover, so I created my script, inspired by the "Rotor in Hover" example.

However, for some reason, the induced velocity is in the wrong direction:
![image](https://github.com/byuflowlab/FLOWUnsteady/assets/90248888/97542885-a7b4-49e5-ba19-54fc8ea1f4c0)

![image](https://github.com/byuflowlab/FLOWUnsteady/assets/90248888/582b279c-3131-43d4-bf20-45dd9960aeef)

The blades are rotating in the correct direction in ParaView.

Why could this be?

I think it could have something do to with the twist (different signs of twist spline and data). In ParaView, the blades' twist angles look ok.
Why could the twist spline have a negative spline?

![image](https://github.com/byuflowlab/FLOWUnsteady/assets/90248888/b901d0ae-f319-407c-9531-d5536e93345d)

**Comment by christianhauschel** [comment]
Actually, I want to simulate a quadcopter.
Here I have the same problem... the induced velocity in the opposite direction of the blade vtk file:

![image](https://github.com/byuflowlab/FLOWUnsteady/assets/90248888/837fb6eb-9202-4d00-97d9-8f5a9e83323d)

![image](https://github.com/byuflowlab/FLOWUnsteady/assets/90248888/f30d1f82-03b6-4416-bcb9-bbbc73c13de7)

I need the specific coordinate convention. For this, I transform the coordinate system by:
```julia
      Oaxis = [
          0.0 0.0 1.0
          0.0 1.0 0.0
          1.0 0.0 0.0
      ]

      vlm.setcoordsystem(rotors[i], Oposition[i, :], Oaxis; user=false)
```

**Comment by EdoAlvarezR** [comment]
Hi Christian!

It's difficult to tell what's happening from the screenshots. Could you share the script that you are using to run the simulation along the files defining the rotor geometry?

**Comment by christianhauschel** [comment]
I have figured it out somehow ;)

**Reply by inse0918** [reply]
how about checking the flip of the propeller ?

**Comment by EdoAlvarezR** [comment]
I'm glad you were able to figure out the issue. What was it?

**Reply by christianhauschel** [reply]
Some combination of the axes and the setting of CW/CCW
